packages/shared/src/components/modals/ShareModal.spec.tsx

```diff
@@ -86,9 +82,10 @@ describe('ShareModal Test Suite:', () => {
     mockWindowOpen.mockClear();
     window.open = origWindowOpen;
   });
+
   it('should render the component without logged in user', async () => {
     renderComponent(false, false);
-    expect(screen.getByText('Share post')).toBeInTheDocument();
+    expect(screen.getByText('Share with your squad')).toBeInTheDocument();
```

Comment:
Why did this change?

Reply:
Some things related to mocking `useViewSize` which is not relevant here anymore.